client/state/preferences/selectors.js

```diff
@@ -33,6 +33,10 @@ export function getPreference( state, key ) {
 	);
 }
 
+export function getAllRemotePreferences( state ) {
```

Comment:
We're missing a docblock comment - which by the way would be very handy to know if this function returns values or key-value pairs.